### PR TITLE
model: fix nan default dm

### PR DESCRIPTION
**Pull request by mloubout**

### CODE REVIEW

File: examples/seismic/model.py [hunk 1]
```diff
@@ -336,7 +336,7 @@ def dm(self):
         as `dm = div(m)`.
```
Comment (FabioLuporini) [inline examples/seismic/model.py:336]: ? div(vp)

File: examples/seismic/model.py [hunk 2]
```diff
@@ -336,7 +336,7 @@ def dm(self):
...
         """
         dm = Function(name="dm", grid=self.grid, space_order=self.space_order)
-        Operator(Eq(dm, div(self.m)))()
+        Operator(Eq(dm, div(self.vp)), subs=self.spacing_map)()
```
Comment (FabioLuporini) [inline examples/seismic/model.py:339]: I guess the fix is self.m -> self.vp ?

Comment (mloubout) [reply examples/seismic/model.py:339]: Yes, `m=1/vp**2` which creates `nan` at the edge( #1250 )

**Comment by mloubout** [comment]
Fixed the text

>Also, nothing needs to be modified where model.dm is called in benchmarks/user/benchmark.py?

No nothing to change there, only issue was that `model.dm` had NaNs due to div(1/vp) at the edge

**Comment by rhodrin** [comment]
Cool, lets merge when green and give `TheMatrix` a spin!

**Pull request merged**